components/timeline/assets/Timeline.scss

```diff
@@ -0,0 +1,60 @@
+@import "@hods/sass-base";
+
+.hods-timeline {
+  overflow: hidden;
+  position: relative;
+
+  &__item {
+    @include govuk-responsive-padding(2, "bottom");
+    padding-left: 22px;
+    position: relative;
+
+    &:hover {
+      background-color: govuk-colour("mid-grey");
+    }
+
+    &:before { /* styling for lines in margin */
+      background-color: govuk-colour("blue");
+      content: "";
+      @include govuk-responsive-padding(1, "top");
+      @include govuk-responsive-padding(3, "right");
+      left: 0;
+      position: absolute;
+      @include govuk-responsive-margin(1, "top");
+      z-index: 1;
+    }
+  }
+
+  &:before { /* styling for lines in margin */
+    background-color: govuk-colour("blue");
+    content: "";
+    height: 100%;
+    left: 0;
+    position: absolute;
+    margin-top: 8px;
```

Comment:
`0.5em`?

Reply:
0.55em works better. Have changed this now